### PR TITLE
Add composer.json.

### PR DESCRIPTION
**Pull request by steveoliver**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "description": "Local images integration for media entity.",
+  "keywords": ["Drupal"],
+  "license": "GPL-2.0+",
+  "homepage": "https://github.com/drupal-media/media_entity_image",
+  "minimum-stability": "dev",
+  "support": {
+    "issues": "https://github.com/drupal-media/media_entity_image/issues",
+    "source": "https://github.com/drupal-media/media_entity_image"
+  },
+  "require": { }
```
Comment (slashrsm) [inline composer.json]: Do we need to list media_entity here?

File: composer.json [hunk 2]
```diff
@@ -10,5 +10,13 @@
     "issues": "https://github.com/drupal-media/media_entity_image/issues",
     "source": "https://github.com/drupal-media/media_entity_image"
   },
-  "require": { }
```
Comment (steveoliver) [inline composer.json]: @slashrsm Requiring media_entity now.

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "drupal/media_entity_image",
+  "type": "drupal-module",
+  "description": "Local images integration for media entity.",
+  "keywords": ["Drupal"],
+  "license": "GPL-2.0+",
+  "homepage": "https://github.com/drupal-media/media_entity_image",
```
Comment (slashrsm) [inline composer.json]: Homepage should be project page on drupal.org.

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  "minimum-stability": "dev",
+  "support": {
+    "issues": "https://github.com/drupal-media/media_entity_image/issues",
+    "source": "https://github.com/drupal-media/media_entity_image"
```
Comment (slashrsm) [inline composer.json]: Should also link to drupal.org counterparts.

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/drupal-media/media_entity"
```
Comment (slashrsm) [inline composer.json]: Same here.

File: composer.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    }
+  ],
+  "require": {
+    "drupal/media_entity": "dev-8.x-1.x"
```
Comment (slashrsm) [inline composer.json:21]: Checked how commerce does this and it seems that we should require core too. Would be also nice to update composer.json in media_entity.

**Comment by slashrsm** [comment]
Found few more things that I missed first time. This should really be it now.

**Comment by steveoliver** [comment]
Made those changes, but damn, shouldn't have rebased! :/

**Comment by slashrsm** [comment]
Rebased, squashed and merged: ce71532

Thanks!

**Comment by steveoliver** [comment]
You're quick! I just updated description and keywords if you think those are helpful. (https://github.com/drupal-media/media_entity_image/pull/12/commits/5fad91227be43c67cbc903d544527fe3bc587681)

**Pull request closed without merging**